managed/models/database.go

```diff
@@ -1253,12 +1250,6 @@ func setupFixture1(q *reform.Querier, params SetupDBParams) error {
 	return nil
 }
 
-func setupFixture2(q *reform.Querier, username, password string) error { //nolint:revive
```

Comment:
Removing it since it's been there for very long, but we can always restore if needed.